package.json

```diff
@@ -86,6 +87,7 @@
     "filed": "^0.1.0",
     "jscs": "^2.7.0",
     "nodeunit": "^0.9.0",
+    "restify-clients": "1.1.0",
```

Comment:
Curious why clients is both a dev and normal dependency, shouldn't normal dependency win out?